backend/x11/backend.c

```diff
@@ -221,11 +221,20 @@ static bool wlr_x11_backend_start(struct wlr_backend *backend) {
 
 	init_atom(x11, &x11->atoms.wm_protocols, 1, "WM_PROTOCOLS");
 	init_atom(x11, &x11->atoms.wm_delete_window, 0, "WM_DELETE_WINDOW");
+	init_atom(x11, &x11->atoms.net_wm_name, 1, "_NET_WM_NAME");
+	init_atom(x11, &x11->atoms.utf8_string, 0, "UTF8_STRING");
 
 	xcb_change_property(x11->xcb_conn, XCB_PROP_MODE_REPLACE, output->win,
 		x11->atoms.wm_protocols.reply->atom, XCB_ATOM_ATOM, 32, 1,
 		&x11->atoms.wm_delete_window.reply->atom);
 
+	char title[32];
+	if (snprintf(title, 32, "wlroots - %s", output->wlr_output.name))
```

Comment:
Add `{}` to this if statement please

Reply:
Also, can you use `sizeof(title)` so that there's no bad surprise if we change it?

Reply:
All updated. Thanks for the review.